Tasks/ServiceFabricUpdateManifests/task.json

```diff
@@ -123,6 +123,14 @@
             "required": true,
             "visibleRule": "updateType = Docker image settings",
             "helpMarkDown": "Path to a text file that contains the digest values of the Docker images associated with the Service Fabric application. This file can be output by the [Docker task](https://go.microsoft.com/fwlink/?linkid=848006) when using the push action. The file should contain lines of text in the format of 'registry/image_name@digest_value'."
+        },
+        {
```

Comment:
To add a new property like this you need to increase the minor version number (new feature that is backwards compatible)